docs/build/spells.md

```diff
@@ -0,0 +1,197 @@
+# Spells
+
+Spells are actors in the Fluence Network designed to perform tasks with user business logic as well as maintenance jobs. 
+
+Think of them as of automated jobs, that can be executed when some specific trigger event occurs (time, connection poll, http, blockchain events, etc).
+
+Spells can be used for user-defined automation scenarios for service maintenance, periodical data extraction, subnet maintenance, etc. 
+
+A spell is a [Marine](/docs/marine-book/introduction.md) service, the main goal of which is to provide state storage for script execution. It also provides some additional features to ease the spell usage for users and take some responsibility from the node (aka [Nox](https://github.com/fluencelabs/nox)). The service serves as a template and is instantiated on the creation of a new spell.
+
+This service is intended to be used as a template and instantiated by a node on each new spell creation.
+
+## Managing spells
+[Nox](https://github.com/fluencelabs/nox) provides user API for interacting with spells.
+
+Users are able to control their spells' lifecycle: install new ones, update them, and remove them. The developer experience is provided by the means of [Fluence CLI](https://github.com/fluencelabs/cli). 
+
+A spell consists of the following parts: 
+- The spell script, which will be executed
+- The list of trigger events (aka the trigger configuration) on which to run the spell
+- The initial state of the spell (e.g. initial state of spell's KV storage)
+
+After installation, the developer is able to update the initial settings: the spell script, the trigger configuration, and the state KV storage.
+
+Also, the developer is able to remove their spells, cleaning the state from the peer.
+
+Developer can check their spells' state: 
+- inspect their internal state (the KV storage, the script, the trigger config)
+- check if they’re currently subscribed to some triggers
```

Comment:
I don't think there's a way to do that. One can deduct it from trigger config, but there are no "unsubscribed" spells.